test/functional/system/transactions/2.delegates/4.different.accounts.different.usernames.js

```diff
@@ -126,22 +126,34 @@ describe('system test (type 2) - double delegate registrations', () => {
 						);
 					});
 
+					// The timeout of 1 second is necessary here because of the way we generate signature
+					// The signatures are generated using timestamp in seconds(not milliseconds)
+					// Due to this when we create transaction within millisecond using same data
+					// The transaction will have same transaction id due to use of seconds
 					it('adding to pool delegate registration with already registered username should fail', done => {
-						localCommon.addTransaction(library, transaction1, err => {
-							expect(err).to.equal(
-								`Transaction is already confirmed: ${transaction1.id}`
-							);
-							done();
-						});
+						setTimeout(() => {
+							const transaction3 = lisk.transaction.registerDelegate({
+								passphrase: account2.password,
+								username: account2.username,
+							});
+							localCommon.addTransaction(library, transaction3, err => {
+								expect(err).to.equal('Account is already a delegate');
+								done();
+							});
+						}, 1000);
 					});
 
 					it('adding to pool delegate registration from same account should fail', done => {
-						localCommon.addTransaction(library, transaction2, err => {
-							expect(err).to.equal(
-								`Transaction is already confirmed: ${transaction2.id}`
-							);
-							done();
-						});
+						setTimeout(() => {
+							const transaction4 = lisk.transaction.registerDelegate({
+								passphrase: account2.password,
+								username: account2.username,
+							});
```

Comment:
@ManuGowda Can't use we `timeOffset` here?

Reply:
@nazarhussain sure we can use, but it's optional I believe.